R/get_about_pages.r

```diff
@@ -48,7 +51,12 @@ extract_about_links <- function(base_url, timeout_thres = 10) {
   if (url.exists(base_url, timeout = timeout_thres) == FALSE) {
     stop(glue("This URL is not responding ({timeout_thres} seconds timeout)."))} 
   
-  # First, try looking for it directly
+  # If base url includes either index.php or index.html removes it 
+  if (str_detect(base_url, "index.php|index.html")) {
```

Comment:
there can be other patterns here as well like index.htm or home.asp. that's why i did that slightly silly thing of looking at whether the last part of the URL has a period in it.

Reply:
And in this case: http://www.ccunm.org/asp/home.asp  the page won't load without home.asp. The about page is at http://www.ccunm.org/ASP/aboutus.asp

Reply:
@milandv Totally makes sense. I will incorporate that part from your PR and will push a new commit.

Reply:
@milandv The new commit solves this problem to some extent but not the full. The remaining problem is `xml2::read_html()` doesn't read ASP file. Do you know how to resolve this?

Reply:
i don't have a general solution but it looks like some are read and some are not, maybe?

this one seems okay: https://vfwri.org/di/vfw/v2/default.asp

Reply:
(In which case I think we just let it fail where it can't read them and succeed otherwise)

Reply:
nevermind. this works for the first one as well:

read_html(GET("http://www.ccunm.org/asp/home.asp")) %>% html_nodes("a") %>% html_attr("href")

(whereas read_html alone does not)

Reply:
@milandv I dug a little deeper ... my current impression is that it's an encoding issue. What the server sent us back is not correctly encoded. If we change the encoding to ISO-8859-1, we get the correct character vector; otherwise, it's just NA.

 Encoding Language Confidence
1 ISO-8859-1       en       0.39
2 ISO-8859-2       ro       0.16
3   UTF-16BE                0.10
4   UTF-16LE                0.10
5 ISO-8859-9       tr       0.10

Reply:
The new commit solved this encoding problem but it didn't entirely resolve the issue as the URL path doesn't form correctly. Hopefully, I will resolve this by this week.